modules/assim.batch/R/pda.emulator.R

```diff
@@ -281,6 +272,8 @@ pda.emulator <- function(settings, params.id = NULL, param.names = NULL, prior.i
           SS.list[[inputi]] <- cbind(X, error.statistics[[inputi]])
       } # if-block
         
+      # remove failed runs
+      SS.list[[inputi]] <- SS.list[[inputi]][!rowSums(is.na(SS.list[[inputi]])), ]
     } # for-loop
```

Comment:
I'd do the check as a fraction and make it user specified (settings). I think a reasonable default if it's not specified might be 50% -- if more than half the runs fail something's probably really wrong.  But I could envision cases where someone might be very strict, and other cases where someone might be proposing very broad parameter that result in many impossible combinations that would require them to be fairly lenient.